robots_test.cc

```diff
@@ -123,7 +123,7 @@ TEST(RobotsUnittest, ID_LineSyntax_Groups) {
   EXPECT_FALSE(IsUserAgentAllowed(robotstxt, "BazBot", url_foo));
 }
 
-// REP lines are case insensitive. See REP I-D section "Protocol Definition".
+// REP lines are case-insensitive. See REP I-D section "Protocol Definition".
```

Comment:
Both case-insensitive and case insensitive are correct. So I prefer to avoid changing it.
```suggestion
// REP lines are case insensitive. See REP I-D section "Protocol Definition".
```